scripts/run_backend_tests.py

```diff
@@ -0,0 +1,93 @@
+# Copyright 2019 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""This script runs all the (Python) backend tests, in parallel."""
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+import argparse
+import os
+import subprocess
+import sys
+
+import python_utils
+
+from . import backend_tests
+from . import build
+from . import common
+from . import install_third_party_libs
+from . import setup
+from . import setup_gae
+
+_PARSER = argparse.ArgumentParser()
+_PARSER.add_argument(
```

Comment:
Where are the other flags?

Reply:
What other flags are we talking about?

Reply:
```bash
# (1) Generate a coverage report by adding the argument	
#	
#   --generate_coverage_report	
#	
# but note that this will slow down the tests by a factor of 1.5 or more.	
#	
# (2) Append a test target to make the script run all tests in a given module	
# or class, or run a particular test. For example, appending	
#	
#   --test_target='foo.bar.Baz'	
#	
# runs all tests in test class Baz in the foo/bar.py module, and appending	
#	
#   --test_target='foo.bar.Baz.quux'	
#	
# runs the test method quux in the test class Baz in the foo/bar.py module.	
#	
# (3) Append a test path to make the script run all tests in a given	
# subdirectory. For example, appending	
#	
#   --test_path='core/controllers'	
#	
# runs all tests in the core/controllers/ directory.	
#	
# (4) Enable the verbose log by add the argument. It will display the outputs of	
#   the tests being run.	
#	
#   --verbose or -v
```

Reply:
Oh they're all still present in scripts/backend_tests.py. This file is run_backend_tests.py

Reply:
Hmm that's really confusing, can we just expand and rename that file instead of introducing an indirection?

Reply:
Done